pivy-importer/src/main/groovy/com/linkedin/python/importer/ImporterCLI.java

```diff
@@ -57,26 +59,45 @@ public static void main(String[] args) throws Exception {
             throw new RuntimeException("Unable to continue, no repository location given on the command line (use the --repo switch)");
         }
         final File repoPath = new File(line.getOptionValue("repo"));
-        final DependencySubstitution replacements = new DependencySubstitution(buildSubstitutionMap(line), buildForceMap(line));
 
         repoPath.mkdirs();
 
         if (!repoPath.exists() || !repoPath.isDirectory()) {
             throw new RuntimeException("Unable to continue, " + repoPath.getAbsolutePath() + " does not exist, or is not a directory");
         }
 
+        importPackages(line, repoPath);
+        logger.info("Execution Finished!");
+    }
 
+    private static void importPackages(CommandLine line, File repoPath) {
+        final DependencySubstitution replacements = new DependencySubstitution(buildSubstitutionMap(line), buildForceMap(line));
         Set<String> processedDependencies = new HashSet<>();
         for (String dependency : line.getArgList()) {
-            DependencyDownloader dependencyDownloader = new DependencyDownloader(
-                    dependency, repoPath, replacements, line.hasOption("latest"), line.hasOption("pre"),
-                    line.hasOption("lenient"));
-            dependencyDownloader.getProcessedDependencies().addAll(processedDependencies);
-            dependencyDownloader.download();
-            processedDependencies.addAll(dependencyDownloader.getProcessedDependencies());
-        }
+            DependencyDownloader artifactDownloader;
```

Comment:
Is there a specific reason you changed the name of this variable to `artifactDownloader`?

Reply:
Since it will be initiated as SdistDownloader or WheelsDownloader later.